cmd/neofs-node/config/node/config.go

```diff
@@ -24,16 +27,40 @@ var (
 // from "node" section.
 //
 // Panics if value is not a non-empty string.
-func Key(c *config.Config) string {
+func Key(c *config.Config) *keys.PrivateKey {
 	v := config.StringSafe(c.Sub(subsection), "key")
 	if v == "" {
 		panic(errKeyNotSet)
 	}
 
-	// TODO: add string -> `ecdsa.PrivateKey` parsing logic
-	// after https://github.com/nspcc-dev/neofs-node/pull/569.
+	var (
+		key  *keys.PrivateKey
+		err  error
+		data []byte
+	)
+	if data, err = ioutil.ReadFile(v); err == nil {
+		key, err = keys.NewPrivateKeyFromBytes(data)
+	}
+
+	if err != nil {
+		panic(fmt.Errorf("can't read key: %w", err))
+	}
+
+	return key
+}
+
+// Wallet returns value of node private key from "node" section.
+func Wallet(c *config.Config) *keys.PrivateKey {
+	v := c.Sub(subsection).Sub("wallet")
+	acc, err := innerringconfig.LoadAccount(
+		config.String(v, "path"),
+		config.String(v, "address"),
+		config.String(v, "password"))
+	if err != nil {
+		return Key(c)
```

Comment:
I am not sure this fallback should be done in config. Maybe `Wallet` can return `nil` if section is empty? 

If you want to keep fallback from one section to another, then I propose to do that in `Key` function. In the end it returns PrivateKey, thus makes more sense.

Reply:
IMO this is exactly what should be done in config: there is somesome info about a key and we just want to retrieve it.
I have put this in `Wallet` for wallet to have priority over raw key. Reversed the order here.